common/src/main/java/org/apache/uniffle/common/ShuffleBlockInfo.java

```diff
@@ -84,6 +86,14 @@ public ShuffleBlockInfo(
     this.taskAttemptId = taskAttemptId;
   }
 
+  public int getRetryCounter() {
+    return retryCounter;
+  }
+
+  public void setRetryCounter(int retryCounter) {
```

Comment:
Do we need this method? `setCounter` seems weird. `increateCounter` or `resetCounter`?

Reply:
Please see the invoking code.